docs/docs/debugging-the-build-process.md

```diff
@@ -6,8 +6,8 @@ Gatsby's `build` and `develop` steps run as a Node.js application which you can
 
 In this guide you will learn how to debug some code using:
 
-- [VS Code debugger](<#vs-code-debugger-(auto-config)>) (Auto-Config)
-- [VS Code debugger](<#vs-code-debugger-(manual-config)>) (Manual-Config)
+- [VS Code debugger](<#vs-code-debugger-auto-config>) (Auto-Config)
```

Comment:
```suggestion
- [VS Code debugger (Auto-Config)](<#vs-code-debugger-auto-config>)
```

Reply:
it's part of the title - so it should also be in link

Reply:
to avoid the 2 lines looks like they point to the same anchor

Reply:
@muescha yeah, agree thanks 👌